### ISSUE TITLE
RetroArch backend

### ISSUE THREAD

**Issue by japhib** [egordorichev/pemsa#12]
It would be cool to compile Pemsa with a RetroArch backend so it could be used on emulators that use RetroArch. There's another PICO-8 core called retro-8 that is available via RetroArch, but it's not fully up-to-date and I believe it doesn't emulate PICO-8 as closely as Pemsa does. For example, it uses floating point numbers rather than fixed-point, and I think some of the newer features are just completely unsupported.

I'm not sure what compiling against RetroArch entails, but I wanted to open an issue since it seems like a cool goal.

**Comment by egordorichev** [comment]
Yes, that would be really cool. I've done some quick googling around, shouldn't retroarch run sdl2 just fine? What is requried to get it running?

**Comment by japhib** [comment]
Not sure, I'll have to take a closer look at RetroArch docs. [retro8](https://github.com/Jakz/retro8) is the other PICO-8 emulator that currently has a RetroArch backend, so it seems like we could also use that as a resource and look how its compilation is set up.
